src/cli.rs

```diff
@@ -109,6 +109,14 @@ pub struct Cli {
     #[clap(short = 'P', long, value_name = "FORMAT")]
     pub history_print: Option<Print>,
 
+    /// Resolve hostname to ipv4 addresses only.
+    #[clap(short = '4', long)]
+    pub ipv4: bool,
+
+    /// Resolve hostname to ipv6 addresses only.
+    #[clap(short = '6', long)]
+    pub ipv6: bool,
```

Comment:
Can you include these flags in `to_curl.rs`, which handles translating xh commands to curl?